File: 백준/Silver/2563. 색종이/색종이.py
import sys
input = sys.stdin.readline

mtx = [[0]*100 for _ in range(100)]

result = 0
for _ in range(int(input())):
    x,y = map(int,input().split())
    for i in range(y, y+10):
        for j in range(x,x+10):
            if not mtx[i][j]:
                mtx[i][j] = 1
                result += 1

print(result)

File: 백준/Silver/2563. 색종이/색종이.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        StringTokenizer st;
        int result = 0;
        boolean[][] mtx = new boolean[100][100];
        for (int i=0; i<n; i++) {
            st = new StringTokenizer(br.readLine());
            int x = Integer.parseInt(st.nextToken());
            int y = Integer.parseInt(st.nextToken());
            for (int y1=y; y1<y+10; y1++) {
                for (int x1=x; x1<x+10; x1++) {
                    if (!mtx[y1][x1]) {
                        result++;
                        mtx[y1][x1] = true;
                    }
                }
            }
        }
        System.out.println(result);
    }
}
